### PR TITLE
Documentation versioning

### PR DESCRIPTION
**Pull request by azhavoro**
Build documentation for major releases

### How has this been tested?
https://azhavoro.github.io/cvat/docs/

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
~~- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly~~
~~- [ ] I have added tests to cover my changes~~
~~- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
~~- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: site/build_docs.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    cwd = repo.working_tree_dir
+    content_loc = os.path.join(cwd, 'site')
+    if not os.path.exists(output_dir):
+        os.makedirs(output_dir)
+
+    generate_versioning_config(os.path.join(cwd, 'site', 'versioning.toml'), (t.name for t in tags))
+    run_hugo(content_loc, output_dir)
+
+    generate_versioning_config(os.path.join(cwd, 'site', 'versioning.toml'), (t.name for t in tags), '/..')
```
Comment (nmanovic) [inline site/build_docs.py:55]: @azhavoro , What is a reason to call the function two times?

**Comment by nmanovic** [comment]
@azhavoro , which code will allow us to switch between version?

File: site/build_docs.py (hunk 1 above)
Comment (azhavoro) [reply site/build_docs.py:55]: Due to structure of documentation need to generate 2 different config files for current version and tagged versions:
```shell
.
├── v1.5.0
├── v1.6.0
├── v...
└── index.html (and other documentation files for develop)

```

**Comment by azhavoro** [comment]
> @azhavoro , which code will allow us to switch between version?

This handled by Docsy: https://www.docsy.dev/docs/adding-content/versioning/

**Review by nmanovic — APPROVED** [review]

**Pull request merged**